<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibrationDialog</class>
 <widget class="QDialog" name="calibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Study details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>500</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QPushButton" name="PB_previous">
       <property name="text">
        <string> Back</string>
       </property>
       <property name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/if_Back_32435.png</normaloff>:/icons/if_Back_32435.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="PB_next">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/if_Next_32435.png</normaloff>:/icons/if_Next_32435.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="PB_clear">
        <property name="text">
         <string>Clear all fields</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="2" column="3">
            <widget class="QCheckBox" name="cb_calibration">
             <property name="text">
              <string>Calibration</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QDoubleSpinBox" name="dSB_volume">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The volume of the enjected radio nuclide. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="LE_studyID">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The ID of the experiment is given automatically by the software and is used for storing it in the database.  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="1">
               <widget class="QRadioButton" name="rb_mCi_2">
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="text">
                 <string>Î¼Ci</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoRepeat">
                 <bool>false</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="rb_mBq_2">
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="text">
                 <string>MBq</string>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Type of study:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QDoubleSpinBox" name="dSB_activity">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The activity as measured after the injection. The value is used to calculate the decay correction. If the value is zero, the decay will not be taken into account. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Comments:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>ul</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Date/time:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>Flood volume:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Radionuclide:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="cb_validation">
             <property name="text">
              <string>Validation</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label1">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="4">
            <widget class="QTextEdit" name="TE_comments">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;Set any comments relevant to this experiment.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QComboBox" name="CB_radioNuclide">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The radio nuclide used in the experiment. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QLabel" name="label_35">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The value of the injected activity as computed after the injection. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Flood activity:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QDateTimeEdit" name="dTime_ingection">
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The injection date/time. Injection time is used to compute the time for the decay correction. Note that in order to account for the decay correction the &amp;quot;Injection activity&amp;quot; must be different than zero.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lb_collimator">
             <property name="text">
              <string>Collimator:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QComboBox" name="cB_collimator"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CB_radioNuclide</tabstop>
  <tabstop>dSB_volume</tabstop>
  <tabstop>dSB_activity</tabstop>
  <tabstop>TE_comments</tabstop>
  <tabstop>PB_clear</tabstop>
  <tabstop>PB_previous</tabstop>
  <tabstop>PB_next</tabstop>
 </tabstops>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
